src/reducers.js

```diff
@@ -1129,6 +1129,17 @@ export const sellItem = (state, { id }, howMany = 1) => {
     : getAdjustedItemValue(valueAdjustments, id)
   const saleIsGarnished = isItemAFarmProduct(item)
   let saleValue = 0
+  let salePriceMultiplier = 1
+
+  if (state.completedAchievements['i-am-rich-3']) {
+    salePriceMultiplier = 1.25
+  } else if (state.completedAchievements['i-am-rich-2']) {
+    salePriceMultiplier = 1.1
+  } else if (state.completedAchievements['i-am-rich-3']) {
+    salePriceMultiplier = 1.05
+  }
+
+  console.log('price multiplier value', salePriceMultiplier)
```

Comment:
I know you're aware of this, I just want to add this comment so we don't forget to remove it before merging! 🙂

Reply:
removing this now